frontend/packages/console-app/src/components/pdb/PDBPage.tsx

```diff
@@ -35,7 +35,7 @@ export const PodDisruptionBudgetsPage: React.FC<PodDisruptionBudgetsPageProps> =
   };
   return (
     <>
-      <ListPageHeader title={showTitle ? t(PodDisruptionBudgetModel.labelPluralKey) : undefined}>
+      <ListPageHeader title={showTitle ? t(PodDisruptionBudgetModel?.labelPluralKey || '') : ''}>
```

Comment:
we know that PodDisruptionBudgetModel.labelPluralKey is always defined because this is a static object, is this possible?

```suggestion
      <ListPageHeader title={showTitle ? t(PodDisruptionBudgetModel!.labelPluralKey)}>
```

Reply:
Upon commit we will get a non null assertion typescript-eslint. Hence avoided that